int32 waypoint_index
geometry_msgs/PoseStamped pose
geometry_msgs/TwistStamped twist
int32 mission_state
int32 lane_number
float64 ryaw
float64 rk
